script.js

```diff
@@ -0,0 +1,908 @@
+    } else if (gameState === 'gameOver') {
+      statsDisplay.innerText = '';
+      createGameStatsDisplay('Please insert \ncredits to deal!');
+      outputMsgDisplay.classList.add('animate__animated');
+      outputMsgDisplay.classList.add('animate__flash');
+    }
+  });
+  buttonsContainer.appendChild(swapBtn);
+};
+
+// Function that refreshes scroll machine display
+const refreshScrollMachineDisplay = () => {
+  const topDisplay = document.querySelector('.combinationsDisplay');
+  topDisplay.innerText = '';
+  generateCombinationsTopDisplay();
+  destroyScrollDisplayMachines();
+  createScrollDisplayMachines();
+};
+// Function that creates the display on the statistics on the gameplay below
+const createGameStatsDisplay = (outputMsg) => {
+  // Display how much credits to play for this game
+  creditsInsertedDisplay = document.createElement('div');
+  creditsInsertedDisplay.innerText = `Insert Credit(s): ${numCreditsInserted}`;
+  creditsInsertedDisplay.setAttribute('id', 'creditsInserted');
+
+  // Display what combination is present
+  outputMsgDisplay = document.createElement('div');
+  outputMsgDisplay.setAttribute('id', 'currentCombi');
+
+  if (!outputMsg) {
+    outputMsgDisplay.innerText = `${nameOfWinCombi}`;
+  } else {
+    outputMsgDisplay.innerText = `${outputMsg}`;
+  }
+
+  // Display how much credits the player has left
+  creditsLeftDisplay = document.createElement('div');
+  creditsLeftDisplay.innerText = `Credits Left: ${creditsLeft}`;
+  creditsLeftDisplay.setAttribute('id', 'availCredits');
+
+  statsDisplay.appendChild(creditsInsertedDisplay);
+  statsDisplay.appendChild(outputMsgDisplay);
+  statsDisplay.appendChild(creditsLeftDisplay);
+};
+
+// Function that creates a 'slotMachine' obj for each column of the display abv
+const createScrollDisplayMachines = () => {
+  combiDisplayCol = document.querySelector('.nameOfCombiDisplay');
+  pOut1Col = document.querySelector('.pCol1');
+  pOut2Col = document.querySelector('.pCol2');
+  pOut3Col = document.querySelector('.pCol3');
+  pOut4Col = document.querySelector('.pCol4');
+  pOut5Col = document.querySelector('.pCol5');
+  combiDisplayMachine = new SlotMachine(combiDisplayCol, { active: 9 });
+  pOut1Machine = new SlotMachine(pOut1Col, { active: 9 });
+  pOut2Machine = new SlotMachine(pOut2Col, { active: 9 });
+  pOut3Machine = new SlotMachine(pOut3Col, { active: 9 });
+  pOut4Machine = new SlotMachine(pOut4Col, { active: 9 });
+  pOut5Machine = new SlotMachine(pOut5Col, { active: 9 });
+};
+
+// Function that destroys each of the 'slotMachine' objs previously created
+const destroyScrollDisplayMachines = () => {
+  combiDisplayCol = null;
+  pOut1Col = null;
+  pOut2Col = null;
+  pOut3Col = null;
+  pOut4Col = null;
+  pOut5Col = null;
+};
+
+// Function that initializes the game with certain displays and cover-cards
+const gameInit = () => {
+  buildUI();
+  shuffledDeck = shuffleCards(makeDeck());
+  createGameStatsDisplay();
```

Comment:
Great work decomposing different UI element creation into different functions!

Reply:
Consider declaring the static HTML elements (if any) directly in the index.html or decomposed static HTML files directly to reduce the logic needed in our JS files.